ui/js/component/fileActions/view.jsx

```diff
@@ -206,6 +277,36 @@ class FileActions extends React.PureComponent {
             outpoint={fileInfo.outpoint}
             title={title}
           />}
+        {modal == "insufficientBalance" &&
```

Comment:
We've also changed the way modals work to use constants. I also think this code was copy and pasted, which is a sign it ought to be refactored. These should probably be moved into `app/view.jsx`.

You can see the FIRST_REWARD modal here for reference https://github.com/lbryio/lbry-app/commit/29553bc391e594c0d918179345cf1e33a38c9c40